### DISCUSSION TITLE
Applying one domain rules to another domain

### DISCUSSION THREAD

**Discussion by furuycom** [uBlockOrigin/uBlock-issues#2601] [Ideas] [answered]
Hi. Is there a way to apply the rules of one domain to another domain?
I think this feature necessary because, for example, eksisozluk.com is unfortunately blocked in Turkey (https://en.wikipedia.org/wiki/Ek%C5%9Fi_S%C3%B6zl%C3%BCk#cite_note-:0-5). To bypass the website block, they redirected eksiozluk.com to eksisozluk2023.com. The website is exactly the same, but no filters are working because the domain name has changed.

**Comment by mapx-** [comment, marked as answer]
the turkish list contains already `eksisozluk2023.com` filters

`TUR: AdGuard Turkish`

**Comment by gorhill** [comment]
> they redirected `ekisozluk.com`

~~I get a parked page for `ekisozluk.com`.~~ I see, typo.

**Comment by krystian3w** [comment]
Rules are for firewall mode.

Even MV3 due we don't write rules in DNR jargon but still in EasyList syntax.
